session/pingpong/consumer_balance_tracker.go

```diff
@@ -197,6 +197,12 @@ func (cbt *ConsumerBalanceTracker) ForceBalanceUpdate(id identity.Identity) uint
 	}
 
 	grandTotal, err := cbt.consumerGrandTotalsStorage.Get(id, cbt.accountantAddress)
+	if err == ErrNotFound {
```

Comment:
If possible it would be safer to use `errors.Is(err, ErrNotFound)` as if Get returns wrapped error this code will not get executed.